src/app/public/testing/auth-http-client/sky-http-interceptor.test-utils.ts

```diff
@@ -0,0 +1,71 @@
+import {
```

Comment:
The `public/testing` folder is meant for code to be used by consumers. This class module is only used internally, so it should be moved to a folder closer where it's used like `/src/app/public/modules/auth-http-client/testing`.

Reply:
:+1: Will do.  I was planning to use these functions for my local routing work as well but I can just do a little more duplication as need like the protocol formatting.